client/multi-currency-setup/wizard/task-item.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { Icon, check } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import WizardTaskContext from '../../additional-methods-setup/wizard/task/context';
+import './task-item.scss';
+
+const WizardTaskItem = ( {
+	children,
+	title,
+	index,
+	className,
+	visibleDescription,
+} ) => {
+	const { isCompleted, isActive } = useContext( WizardTaskContext );
+
+	return (
+		<li
+			className={ classNames( 'wcpay-wizard-task', className, {
+				'is-completed': isCompleted,
+				'is-active': isActive,
+			} ) }
+		>
+			<div className="wcpay-wizard-task__top-border" />
+			<div
+				className="wcpay-wizard-task__headline"
+				// tabindex with value `-1` is necessary to programmatically set the focus
+				// on an element that is not interactive.
+				tabIndex="-1"
+			>
+				<div className="wcpay-wizard-task__icon-wrapper">
+					<div className="wcpay-wizard-task__icon-text">
+						{ index }
+					</div>
+					<Icon
+						icon={ check }
+						className="wcpay-wizard-task__icon-checkmark"
+					/>
+				</div>
+				<span className="wcpay-wizard-task__title">{ title }</span>
+			</div>
+			{ visibleDescription && ! isActive && (
+				<span
+					className={ classNames(
+						'wcpay-wizard-task__visible-description-element',
+						'is-muted-color'
+					) }
+				>
+					{ visibleDescription }
+				</span>
+			) }
```

Comment:
Seeing that the only real change to the `additional-methods-setup` on is this part, and `task-item.scss` is almost a copy. What do you think about moving all the wizard components out of both setups and merging them?

You can easily add this change without interfere, and we can avoid creating new components and styles for it. And in the future might be useful is any other wizard should be created.

Reply:
I thought of that, and decided not to do that inside this PR, I'll create another PR to merge this with the `additional-methods-setup` wizard component, because it'll get lost in terms of visibility, if I'll do that in this PR. It'll be better explain why this change is needed on the source component in a new issue, and update the code accordingly. Does it make sense?

Reply:
Addressed in fa0fe15. Thanks for the review!